nextcord/client.py

```diff
@@ -2089,6 +2089,7 @@ async def process_application_commands(self, interaction: Interaction) -> None:
                 _log.debug(
                     "nextcord.Client: Calling your application command now %s", app_cmd.error_name
                 )
+                self.dispatch("on_application_command", interaction, app_cmd)
```

Comment:
Won't this mean the event being dispatched is actually `on_on_application_command`?